client/my-sites/activity/activity-log-item/index.jsx

```diff
@@ -235,6 +238,53 @@ class ActivityLogItem extends Component {
 
 	performCloneAction = () => this.props.cloneOnClick( this.props.activity.activityTs );
 
+	restoreTrashedPost = () => {
+		const { postId } = this.props.activity.activityMeta;
+		this.setState( { canRestoreSingleItem: false } );
+		this.props.restorePost( postId );
+	};
+
+	restoreTrashedComment = () => {
+		const { postId, commentId } = this.props.activity.activityMeta;
+		this.setState( { canRestoreSingleItem: false } );
+		this.props.restoreComment( postId, commentId );
+	};
+
+	renderRestoreAlternatives() {
+		const { activity, canRestoreSingleItem, translate } = this.props;
+
+		switch ( activity.activityName ) {
+			case 'post__trashed':
+				return (
+					<Fragment>
+						<PopoverMenuItem
+							icon="history"
+							onClick={ this.restoreTrashedPost }
+							disabled={ ! this.state.canRestoreSingleItem }
+						>
+							{ translate( 'Restore post' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **33** times:
`translate( 'Restore Post' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).